TTS/tts/datasets/formatters.py

```diff
@@ -102,7 +102,7 @@ def ljspeech(root_path, meta_file):
         for line in ttf:
             cols = line.split("|")
             wav_file = os.path.join(root_path, "wavs", cols[0] + ".wav")
-            text = cols[1]
+            text = cols[2]
```

Comment:
@erogol , this part of changing the cols[1] to cols[2] is giving me an index error . Noticed it today and its due to the recent merge . With the previous version I was not able to get that error and when I change this value back my code seems to run again well

Reply:
I guess you don't use ljspeech dataset but your custom dataset right ?

Reply:
If it is the case, the formatter is for LJSpeech dataset and you should implement your own formatted or add a new column to your dataset's metadata

Reply:
yes am using my own custom dataset. So my metadata is a csv file.   Thank you. My question maybe would be why the change to 2 since initially it still worked with the LJSpeech dataset. Is there some data that was not being captured initially on LJSpeech

Reply:
Because ljspeech has normalized transcripts in the last column and I get better results with it.

Reply:
ohh i get. Thanks alot

Reply:
np :) sorry for the inconvenience on your end